deps/valgrind/valgrind.h

```diff
@@ -6477,8 +6477,8 @@ VALGRIND_PRINTF_BACKTRACE(const char *format, ...)
                                     pool, 0, 0, 0, 0)
 
 /* Associate a piece of memory with a memory pool. */
-#define VALGRIND_MEMPOOL_ALLOC(pool, addr, size)                  \
-    VALGRIND_DO_CLIENT_REQUEST_STMT(VG_USERREQ__MEMPOOL_ALLOC,    \
+#define VALGRIND_MEMSMALL_ALLOC(pool, addr, size)                  \
```

Comment:
As there are so many memory pool related names in here, maybe we should leave Valgrind out of the picture.